powertools-parameters/powertools-parameters-tests/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Some of the tests cut across multiple modules, and have been moved into their own project to facilitate this.